### PR TITLE
Modified Test Case to include check for Memory and MemorySwap

### PR DESCRIPTION
**Pull request by rajdeepd**
Signed-off-by: Rajdeep Dua dua_rajdeep@yahoo.com

### CODE REVIEW

**Comment by cpuguy83** [comment]
Tests failing, looks like you are checking the wrong fields, should be `HostConfig.Memory`, for example.

**Comment by rajdeepd** [comment]
@cpuguy83 fixed

File: integration-cli/docker_api_containers_test.go [hunk 1]
```diff
@@ -853,6 +853,17 @@ func (s *DockerSuite) TestContainerApiPostCreateNull(c *check.C) {
 	if out != "" {
 		c.Fatalf("expected empty string, got %q", out)
 	}
+
+	out_memory, err_memory := inspectField(container.Id, "HostConfig.Memory")
```
Comment (cpuguy83) [inline integration-cli/docker_api_containers_test.go]: Also sorry I didn't mention it before, but please use camel case for variable names.

Comment (rajdeepd) [inline integration-cli/docker_api_containers_test.go]: you mean outMemory and errMemory instead of out_memory

Comment (runcom) [inline integration-cli/docker_api_containers_test.go]: Yup!

**Comment by crosbymichael** [comment]
LGTM

**Comment by runcom** [comment]
LGTM

**Pull request merged**